pkg/kwokctl/runtime/kind/cluster.go

```diff
@@ -52,6 +52,25 @@ func NewCluster(name, workdir string) (runtime.Runtime, error) {
 	}, nil
 }
 
+// Available  checks whether the runtime is available.
+func (c *Cluster) Available(ctx context.Context) error {
+	// kind depends on docker or podman
+	// kwokctl will download kind binary if it is not available
+	// TODO: nerdctl kind provider support is discussing in
+	// https://github.com/kubernetes-sigs/kind/issues/2317 and
+	// https://github.com/containerd/nerdctl/issues/349
+	err := exec.Exec(ctx, "", exec.IOStreams{}, "docker")
+	if err == nil {
+		return nil
+	}
+	err = exec.Exec(ctx, "", exec.IOStreams{}, "podman")
+	if err == nil {
+		return nil
+
+	}
+	return fmt.Errorf("docker or podman is required for kind")
```

Comment:
I thought about this or not to check the more appropriate

```suggestion
	return nil
```